plugin/src/main/groovy/com/facebook/testing/screenshot/build/ScreenshotsPlugin.groovy

```diff
@@ -119,12 +130,16 @@ class ScreenshotsPlugin implements Plugin<Project> {
     }
 
     project.task("recordMode") {
+      group = project.screenshots.GROUP
+      description = "Run all the screenshot tests and record all the screenshots in your screenshots folder"
       doLast {
         recordMode = true
       }
     }
 
     project.task("verifyMode") {
+      group = project.screenshots.GROUP
+      description = "Runs all the screenshot tests and compares it against the previously recorded screenshots"
```

Comment:
Lets keep the verb tense consistent `Runs` -> `Run`, `compares` -> `compare`

`"Run all screenshot tests and compare them against previously recorded screenshots"`